### PR TITLE
Added Navigation and Updated Module

### PR DESCRIPTION
**Pull request by cs-rohit-khune**
Mantis #0794472
- Modified "Employee Watchlist" module, added 'Employee Name' as default grid column
- Removed unnecessary tags

Mantis #0793845
- Added  Sensitive Files & Employee Watchlist in navigation under 'Resource' menu

Updated info.json to the latest format

### CODE REVIEW

File: info.json [hunk 1]
```diff
@@ -81,19 +86,29 @@
...
-        }
+        "recordSets": [
+            {
+                "name": "Employee Watchlist",
+                "apiName": "employeewatchlist",
+                "count": 1
+            },
+            {
+                "name": "Scenario",
+                "apiName": "scenario",
```
Comment (cs-suyog-jinturkar) [inline info.json:104]: Why we added Scenario module?

File: info.json [hunk 2]
```diff
@@ -60,6 +60,11 @@
...
             ]
         },
+        "views": {
+            "navigation": [
+                "Resources"
+            ]
+        },
         "roles": [
             {
                 "name": "Full App Permissions"
```
Comment (cs-suyog-jinturkar) [inline info.json:70]: Do we really need full app permission?

File: views/navigation.json [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    "@type": "SystemViewTemplate",
+    "type": "app",
+    "config": {
+        "navigation": [
+            {
+                "title": "Resources",
+                "icon": "icon icon-resources",
+                "items": [
+                    {
+                        "title": "Attachments",
```
Comment (cs-suyog-jinturkar) [inline views/navigation.json:12]: Why Attachment added here?

File: views/navigation.json [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+                        "state": {
+                            "name": "main.modules.list",
+                            "parameters": {
+                                "module": "attachments"
+                            }
+                        },
+                        "editMode": false
+                    },
+                    {
+                        "title": "Email Templates",
```
Comment (cs-suyog-jinturkar) [inline views/navigation.json:27]: Same here as above

File: info.json (hunk 1 above)
Comment (cs-rohit-khune) [reply info.json:104]: Please note that this is not an entry for the scenario module but it is the entry for scenario record.

File: info.json (hunk 2 above)
Comment (cs-rohit-khune) [reply info.json:70]: Yes, we have provided selective permission on the "Employee Watchlist" and "Sensitive Files" modules under the 'Full app permission role'.

File: views/navigation.json (hunk 3 above)
Comment (cs-rohit-khune) [reply views/navigation.json:12]: For navigation, it does not append a new entry but overwrites the existing one. So we have to keep existing entries as well. In this case 'Attachment' and 'Email Templates'

File: views/navigation.json (hunk 4 above)
Comment (cs-rohit-khune) [reply views/navigation.json:27]: For navigation, it does not append a new entry but overwrites the existing one. So we have to keep existing entries as well. In this case 'Attachment' and 'Email Templates'

**Pull request merged**